driver/controller.go

```diff
@@ -333,8 +333,7 @@ func (d *controllerService) ControllerUnpublishVolume(ctx context.Context, req *
 
 	op, err := client.DetachBlockStorageVolume(ctx, volumeID)
 	if err != nil {
-		if errors.Is(err, v3.ErrNotFound) ||
-			(errors.Is(err, v3.ErrInvalidRequest) && strings.Contains(err.Error(), "Volume not attached")) {
+		if errors.Is(err, v3.ErrNotFound) || strings.Contains(err.Error(), "Volume not attached") {
```

Comment:
Are you now aware of which HTTP code the API is returning? `ErrInvalidRequest ` before was a code range.
Anyway, just checking the error text should be enough for now.

Reply:
I don't know. IMO the correct one for the API to return is ErrNotFound, if it's not that the API is incorrect IMO. But for user friendliness, let's keep the check on the error string as a fallback. Should be good.